docs/blog/2024-07-25-nx-19-5-update.md

```diff
@@ -0,0 +1,117 @@
+
+```json
+{
+  "targetDefaults": {
+    "test": {
+      "dependsOn": ["build"]
+    }
+  }
+}
+```
+
+By adding pattern matching to `targetDefault`s, we can support a wider range of target names, which fits in really well with recent enhancements from Project Crystal, like task atomization. For example, the `@nx/playwright` plugin will automatically add in a task with the name `e2e-ci--${testFileName}` for every playwright spec file in your project. With this new pattern support in `targetDefaults`, we can define default behavior for all these tasks.
+
+The following will set all such tasks to depend on their `build` task:
+
+```json
+{
+  "targetDefaults": {
+    "e2e-ci--**/*": {
+      "dependsOn": ["build"]
+    }
+  }
+}
+```
+
+## NEW: Individual Targets Can Now Opt Out of Parallelism
+
+The ability to "Atomize" larger tasks by automatically breaking them up into smaller pieces is a really cool enhancement that is enabled by Nx Project Crystal. Especially when combined with the ability to distribute tasks across many machines by using Distributed Task Execution from Nx Agents.
+
+{% youtube
+src="https://youtu.be/_FSHQIwITic"
+title="Nx Agents"
+width="100%" /%}
+
+This approach can break down a massive and long-running end-to-end test suite into many smaller tasks. Then you can divide the time it takes to run the entire suite by the number of machines provided via Nx Agents.
+
+However, previously when using this approach, there were some limitations regarding parallelism. Most end-to-end tests cannot run on the same machine at the same time due to shared ports, conflicting database states, etc.
+
+The solution in the past was to add a `--parallel=1` option, which was a bit clunky. For example:
+
+```terminal
+> nx run-many --targets=lint,test,e2e,build --parallel=1
+```
+
+This command above is used to run all lint, test, e2e, and build tasks present in your monorepo. The `--parallel=1` option is needed in order for your e2e tasks to work properly (so as to avoid the problems of running multiple end-to-end tests on the same machine at one time).
+
+Howeve in this scenario we'd also be limiting the parallelism of all our other tasks - which are fine to run in parallel on the same machine.
+
+To address this, all tasks now support a `parallelism` property. By setting this property to `false`, you can now inform the Nx task runner to not attempt to run a certain task in parallel. For example setting this in your `targetDefaults` will mark all atomized e2e tasks in to not run in parallel:
+
+```json
+{
+  // ...
+  "targets": {
+    "e2e-**/*": {
+      // ...
+      "parallelism": false
+    }
+  }
+}
+```
+
+This way we can simply run the command:
+
+```terminal
+> nx run-many --targets=lint,test,e2e,build
```

Comment:
same here, should be `shell`.

Also the command should be

```
nx run-many --targets=lint,test,build,e2e-ci
```

(you can double check in the CI config we generate)